frontend/src/components/patient/CreatePatientForm.js

```diff
@@ -383,12 +383,14 @@ function CreatePatientForm(props) {
                       value={values.subjectNumber || ""}
                       name={field.name}
                       labelText={intl.formatMessage({
-                        id: "patient.subject.number",
+                        id: "patient.first.name",
```

Comment:
Why did you have to switch this code??

Reply:
I will fix this thing

Reply:
I restored the old version:
https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/930/commits/b26086aaf73f308d2d90374b1aa963bef5d601b2